Xamarin.Forms.Core/FontImageSource.cs

```diff
@@ -0,0 +1,61 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Text;
+using System.Threading.Tasks;
+using Xamarin.Forms.Internals;
+using ValidateValueDelegate = Xamarin.Forms.BindableProperty.ValidateValueDelegate;
+
+namespace Xamarin.Forms
+{
+	public partial class FontImageSource : ImageSource
+	{
+		private static BindableProperty CreateBindableProperty<T>(
+			string name,
+			T defaultValue = default(T),
+			Func<T, bool> validateValue = null)
+		{
+			ValidateValueDelegate validate = null;
+			if (validateValue != null)
+				validate = (bindable, value) => validateValue((T)value);
+
+			return BindableProperty.Create(
+				propertyName: name,
+				returnType: typeof(T),
+				declaringType: typeof(FontImageSource),
+				defaultValue: defaultValue,
+				validateValue: validate
+			);
+		}
+
+		public double Size { get => (double)GetValue(SizeProperty); set => SetValue(SizeProperty, value); }
+		public static readonly BindableProperty SizeProperty = CreateBindableProperty(nameof(Size), 30d);
+
+		public char Glyph { get => (char)GetValue(GlyphProperty); set => SetValue(GlyphProperty, value); }
```

Comment:
The one issue I have with this is the fact that UTF-32 (such as unicode emojis) take up 2 characters, so this might be better as a string...